apps/gro/steps.js

```diff
@@ -133,7 +133,94 @@ module.exports = {
     template: 'email',
     fields: ['email-text'],
     continueOnEdit: true,
-    next: '/post'
+    next: '/country'
+  },
+  '/country': {
+    fields: [
+      'country'
+    ],
+    forks: [{
+      target: '/address-outside',
+      condition: function checkIfNotUK(req) {
+        return req.form.values.country !== 'United Kingdom';
+      }
+    }],
+    continueOnEdit: true,
+    next: '/postcode'
+  },
+  '/postcode': {
+    template: 'postcode',
+    fields: [
+      'postcode'
+    ],
+    forks: [{
+      target: '/address-inside',
+      condition: function checkNI(req) {
+        return _.startsWith(req.form.values.postcode, 'BT');
+      }
+    }],
+    continueOnEdit: true,
+    next: '/address-start'
+  },
+  '/address-start': {
+    controller: require('./controllers/address-start'),
+    fields: [
+      'address-found'
+    ],
+    forks: [{
+      target: '/no-postcode',
+      condition: function hasPostcode(req) {
+        return !(req.sessionModel.get('postcode-found'));
```

Comment:
but it would make more sense to save the addresses against the `address` field (if address is a field). You could then use `req.sessionModel.get('addresses`).length > 0`